extension-manifest-v3/package.json

```diff
@@ -4,11 +4,12 @@
   "version": "10.2.15",
   "type": "module",
   "scripts": {
-    "postinstall": "npm run download-engines && npm run download-whotracksme-bloomfilter",
```

Comment:
this change makes download a part of the build. If we want to skip the download, it's probably best to check checksums of the local resources against the remote ones. Let me know if you feel this should be addressed in this PR.